python/python/glide/async_commands/core.py

```diff
@@ -766,6 +766,29 @@ async def lpush(self, key: str, elements: List[str]) -> int:
             int, await self._execute_command(RequestType.LPush, [key] + elements)
         )
 
+    async def lpushx(self, key: str, elements: List[str]) -> int:
+        """
+        Inserts specified values at the head of the `list`, only if `key` already exists and holds a list.
```

Comment:
```suggestion
        Inserts specified values at the head of the list stored at `key`, only if `key` already exists and holds a list.
```

Reply:
tell me if that sounds better:
```suggestion
        Inserts all the specified values at the head of the list stored at `key`, only if the list already exists.
```

Reply:
I personally like the second one so it's a bit similar to lpush documentation

Reply:
I like second option too.
Do you mind if I do another PR to update docs for python and java both? Currect doc strings are copied from java client - we should keep them consistent.

Reply:
both options (adding it now or in a different pr) are fine, maybe it would be even better to add it now so there wont be a confusion on what change needs to be done:)